android/app/src/main/java/suraj/tiwari/reactnativefbads/NativeAdManager.java

```diff
@@ -106,6 +106,7 @@ public void onAdsLoaded() {
   @Override
   public void onAdError(AdError adError) {
     // @todo handle errors here
+    this.sendAppEvent("onAdError", String.format("%d: %s", adError.getErrorCode(), adError.getErrorMessage()));
```

Comment:
Please add the third Locale argument to the String.format call. Various locales handle the %d formatting differently (for instance, devices in the Arabic language will use Arabic numerals instead of standard ones).

It's probably best to force the generic `Locale.ENGLISH`.